### DISCUSSION TITLE
Add IOS & Android files in Keycloak - /.well-known/assetlinks.json

### DISCUSSION THREAD

**Discussion by rkkeycloak** [keycloak/keycloak#17361] [Q&A]
Hi Team,

We are using Keycloak 15

How can we add the IOS and Android files on Keycloak at the location https://{website_url}**/.well-known**/{ios-file}

For example we need to add files here: 

**https://{website_url}/.well-known/apple-app-site-association 
http://{website_url}/.well-known/assetlinks.json**

We do not want /auth/realms/provider/ in the file path. 

Thanks. 
RK

**Comment by darius-m** [comment]
The easiest option is likely to use a reverse proxy, like nginx, and use different `location` blocks for the `/.well-known` files and `/auth`.
